project/MacrosBuild.scala

```diff
@@ -62,10 +61,23 @@ object ScalamacrosBuildPlugin extends AutoPlugin {
     )
   }
 
+  lazy val defaultCompilerVersion = sys.env.get("SCALA_VERSION").fold(scala212) {
+    case "2.12" => scala212
+    case "2.13" => scala213
+    case "dotty" => dotty
+    case unsupported =>
+      throw new IllegalArgumentException(
+        s"SCALA_VERSION=$unsupported. Expected one of: dotty, 2.12, 2.13"
+      )
+  }
+
   override def globalSettings: Seq[Def.Setting[_]] = List(
-    scalaVersion := scala212,
-    crossScalaVersions := List(scala212),
+    scalaVersion := defaultCompilerVersion,
+    crossScalaVersions := List(scala212, scala213, dotty),
     crossVersion := CrossVersion.binary,
+    commands += Command.command("ci-test") { s =>
+      s"plz $defaultCompilerVersion test" :: s
+    },
```

Comment:
@olafurpg could you give a clue what the code is supposed for?

Reply:
It's to only test projects with SCALA_VERSION defined in crossScalaVersions. In sbt 1.0, this can be replaced with `++$SCALA_VERSION test` but with 0.13 we need to use sbt-doge to accomplish the same thing.

Reply:
currently, core is built against 2.12/2.13/dotty while scalac-engine/plugin are 2.12.3 only. If we add a dotty-engine then we can say `plz 0.3.0-RC2 test` to only run `test` for projects that cross-build against dotty.